crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -1209,7 +1209,7 @@ where
             space!()
         }
 
-        if n.computed {
+        if n.key.is_computed() {
```

Comment:
You can replace this with `emit!(n.key)`.
The emit method for `PropName` is aware of `Computed`.